[BUG]: App Crashes when we call JitsiMeet.closeMeeting()

**Platform: Android
Device Physical or Simulator:Physical
Device Model: REDMI NOTE 9 PRO
Pluggin: jitsi_meet: ^2.0.0**

E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.getJitsiView(JitsiMeetActivity.java:110)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.leave(JitsiMeetActivity.java:126)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.finish(JitsiMeetActivity.java:99)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.onConferenceTerminated(JitsiMeetActivity.java:223)
E/JitsiMeetSDK(30570): 	at com.gunschu.jitsi_meet.JitsiMeetPluginActivity.onConferenceTerminated(JitsiMeetPluginActivity.kt:88)
E/JitsiMeetSDK(30570): 	... 11 more
E/JitsiMeetSDK(30570):
E/JitsiMeetSDK(30570): java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.ListenerUtils.runListenerMethodOnUiThread(ListenerUtils.java:139)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.ListenerUtils.access$000(ListenerUtils.java:35)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.ListenerUtils$1.run(ListenerUtils.java:114)
E/JitsiMeetSDK(30570): 	at android.os.Handler.handleCallback(Handler.java:883)
E/JitsiMeetSDK(30570): 	at android.os.Handler.dispatchMessage(Handler.java:100)
E/JitsiMeetSDK(30570): 	at android.os.Looper.loop(Looper.java:224)
E/JitsiMeetSDK(30570): 	at android.app.ActivityThread.main(ActivityThread.java:7562)
E/JitsiMeetSDK(30570): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK(30570): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
E/JitsiMeetSDK(30570): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
E/JitsiMeetSDK(30570): Caused by: java.lang.reflect.InvocationTargetException
E/JitsiMeetSDK(30570): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.ListenerUtils.runListenerMethodOnUiThread(ListenerUtils.java:135)
E/JitsiMeetSDK(30570): 	... 9 more
E/JitsiMeetSDK(30570): **Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'org.jitsi.meet.sdk.JitsiMeetView org.jitsi.meet.sdk.JitsiMeetFragment.getJitsiView()' on a null object reference**
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.getJitsiView(JitsiMeetActivity.java:110)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.leave(JitsiMeetActivity.java:126)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.finish(JitsiMeetActivity.java:99)
E/JitsiMeetSDK(30570): 	at org.jitsi.meet.sdk.JitsiMeetActivity.onConferenceTerminated(JitsiMeetActivity.java:223)
E/JitsiMeetSDK(30570): 	at com.gunschu.jitsi_meet.JitsiMeetPluginActivity.onConferenceTerminated(JitsiMeetPluginActivity.kt:88)
E/JitsiMeetSDK(30570): 	... 11 more
I/Process (30570): Sending signal. PID: 30570 SIG: 9
Lost connection to device.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -50,5 +50,5 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     // Jitsi Meet from https://github.com/jitsi/jitsi-maven-repository/tree/master/releases/org/jitsi/react/jitsi-meet-sdk
-    implementation ('org.jitsi.react:jitsi-meet-sdk:2.11.0') { transitive = true }
+    implementation ('org.jitsi.react:jitsi-meet-sdk:3.2.0') { transitive = true }
 }

--- a/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt
+++ b/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt
@@ -69,19 +69,19 @@ class JitsiMeetPluginActivity : JitsiMeetActivity() {
         registerReceiver(myReceiver, IntentFilter(JITSI_MEETING_CLOSE))
     }
 
-    override fun onConferenceWillJoin(data: MutableMap<String, Any>?) {
+    override fun onConferenceWillJoin(data: HashMap<String, Any>) {
         Log.d(JITSI_PLUGIN_TAG, String.format("JitsiMeetPluginActivity.onConferenceWillJoin: %s", data))
         JitsiMeetEventStreamHandler.instance.onConferenceWillJoin(data)
         super.onConferenceWillJoin(data)
     }
 
-    override fun onConferenceJoined(data: MutableMap<String, Any>?) {
+    override fun onConferenceJoined(data: HashMap<String, Any>) {
         Log.d(JITSI_PLUGIN_TAG, String.format("JitsiMeetPluginActivity.onConferenceJoined: %s", data))
         JitsiMeetEventStreamHandler.instance.onConferenceJoined(data)
         super.onConferenceJoined(data)
     }
 
-    override fun onConferenceTerminated(data: MutableMap<String, Any>?) {
+    override fun onConferenceTerminated(data: HashMap<String, Any>) {
 
         Log.d(JITSI_PLUGIN_TAG, String.format("JitsiMeetPluginActivity.onConferenceTerminated: %s", data))
         JitsiMeetEventStreamHandler.instance.onConferenceTerminated(data)

